readme.md

```diff
@@ -482,6 +482,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Web Design](https://github.com/nicolesaidy/awesome-web-design) - For digital designers.
 - [JMeter](https://github.com/aliesbelik/awesome-jmeter) - Load testing and performance measurement tool.
 - [Creative Coding](https://github.com/terkelg/awesome-creative-coding) - Programming something expressive instead of something functional.
+- [No Login Web Apps](https://github.com/aviaryan/awesome-no-login-web-apps) - Web apps that work without login
```

Comment:
Should it be `No-Login Web Apps`? I'm not sure, but sounds better. You're also missing a `.` at the end.